src/views/v3/Bridge/Routes/details/index.ts

```diff
@@ -0,0 +1,4 @@
+import MaxSlippage from './MaxSlippage';
```

Comment:
You can use the `export from` syntax btw

`export { default as MaxSlippage } from "./MaxSlippage"`